frontend/src/components/HigherOrder/withAuth.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import SideBar from "../SideBar";
+import {Redirect} from "react-router-dom";
+import Cookies from 'js-cookie';
+import store from '../../store/index.js';
+
+export default function withAuth(Component, navId) {
+    return class extends Component {
+        render() {
+            let isLoggedIn = store.getState().loginReducers.isLoggedIn || Cookies.get("userPass");
```

Comment:
use a const if you are not gonna reassign this let. That way people know that this isn't meant to be reinitialized to another value.

Reply:
done